### PR TITLE
Pet collar sounds

### PR DESCRIPTION
**Pull request by Darkstick**
### Intent of your Pull Request

I wanted collars to make little bell noises to maximize puppy realism. Also works on people.

Mechanically similar to clown shoes, a bell noise plays every time whatever's wearing the collar takes 3 steps.

#### -download links to both sound effect used-
https://cdn.discordapp.com/attachments/326011850309500928/696182583985176606/collarbell1.ogg
https://cdn.discordapp.com/attachments/326011850309500928/696182615094329414/collarbell2.ogg

#### Changelog

:cl:  
soundadd: collar bell noises
/:cl:

### CODE REVIEW

**Comment by Marmio64** [comment]
Degenerates like you belong on a cross.

**Comment by Readystorm546** [comment]
" Also works on people."

**Comment by overlord12345** [comment]
There is no bell on the collar very unrealistic

**Review by Hopekz — APPROVED** [review]
nice

**Review by Hopekz — APPROVED** [review]
After changes

**Comment by athena413** [comment]
uh oh

File: code/modules/mob/living/simple_animal/friendly/pet.dm [hunk 1]
```diff
@@ -23,6 +23,16 @@
...
 
+/mob/living/simple_animal/pet/Move(NewLoc, direct)
+	. = ..()
+
+	if(pcollar)
+		if(mobility_flags)
+			if(loc == NewLoc)
+				if(!has_gravity(loc))
+					return
+				pcollar.step_action()
```
Comment (alexkar598) [inline code/modules/mob/living/simple_animal/friendly/pet.dm]: Early returns are generally prefered, also whats up with the mobility_flags there?
```suggestion
	if(!pcollar)
		return
	if (!(mobility_flags & MOBILITY_STAND))
		return
	if(loc != NewLoc)
		return
	if(!has_gravity(loc))
		return
		
	pcollar.step_action()
```

**Review by alexkar598 — CHANGES_REQUESTED** [review]

**Comment by alexkar598** [comment]
The sound number 2 is kinda weird IMO when you listen to it by itself

**Comment by HomocidlePancakes** [comment]
I disapprove.

**Pull request merged**